ChartsDemo-iOS/Objective-C/Demos/PieChartViewController.m

```diff
@@ -28,7 +28,7 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    self.title = @"Pie Bar Chart";
```

Comment:
@petester42 @jjatie need some advise on the name. Pie Bar Chart is ok or kind of typo ?

Reply:
IMO
"Pie Chart" is better.

Reply:
I would agree. It's more clear.